server/src/test/java/org/apache/druid/server/coordinator/rules/PeriodLoadRuleTest.java

```diff
@@ -186,4 +192,69 @@ public void testMappingNull() throws Exception
     Assert.assertEquals(expectedPeriodLoadRule.getPeriod(), inputPeriodLoadRule.getPeriod());
     Assert.assertEquals(expectedPeriodLoadRule.isIncludeFuture(), inputPeriodLoadRule.isIncludeFuture());
   }
+
+  @Test(expected = IAE.class)
+  public void testCreatingNegativeTieredReplicants()
+  {
+    new PeriodLoadRule(Period.days(1), true, ImmutableMap.of(DruidServer.DEFAULT_TIER, -1), true);
+  }
+
+  @Test(expected = IAE.class)
```

Comment:
Use `Assert.assertThrows` instead and verify the exception message too.

Reply:
Changed